pkg/controller/controller_clusterservicebroker.go

```diff
@@ -138,6 +138,7 @@ func (c *controller) updateClusterServiceBrokerClient(broker *v1beta1.ClusterSer
 		return nil, err
 	}
 	clientConfig := NewClientConfigurationForBroker(broker.ObjectMeta, &broker.Spec.CommonServiceBrokerSpec, authConfig)
+	clientConfig.TimeoutSeconds = int(c.OSBAPITimeOut.Seconds())
```

Comment:
Can you move this to `NewClientConfigurationForBroker()`?

Reply:
@luksa Yes, if so, I need to change the NewClientConfigurationForBroker method. I will recheck it. Thanks for review.